### PR TITLE
Welcome

### PR DESCRIPTION
**Pull request by weeryan17**
- [x] Completed PR - Has all features or fixes that it was meant to and isn't half done
- [x] Follows the style guide - Style guide found [here](https://github.com/FlareBot/FlareBot/blob/master/CONTRIBUTING.md)
- [x] Your code follows the same code style as the rest of the project
- [x] Tested changes

## What is added
N/A

## What is fixed
N/A

## What was changed.
I redid welcome to include dm welcomes and also add multiple messages. Also redid the welcome object to suit the new welcome.

### CODE REVIEW

**Comment by ArsenArsen** [comment]
Welcome!

File: src/main/java/stream/flarebot/flarebot/commands/administrator/WelcomeCommand.java [hunk 1]
```diff
@@ -93,4 +186,30 @@ public CommandType getType() {
...
+
+    private void sendWelcomeTable(List<String> messages, int page, TextChannel channel) {
+        int pages = messages.size() < messagesLength ? 1 : (messages.size() / messagesLength) + (messages.size() % messagesLength != 0 ? 1 : 0);
+        int start = messagesLength * (page - 1);
+        int end = Math.min(start + messagesLength, messages.size());
+        if (page > pages || page < 0) {
+            MessageUtils.sendErrorMessage("That page doesn't exist. Current page count: " + pages, channel);
+        } else {
+            List<String> messagesSub = messages.subList(start, end);
+            List<List<String>> body = new ArrayList<>();
```
Comment (ArsenArsen) [inline src/main/java/stream/flarebot/flarebot/commands/administrator/WelcomeCommand.java]: ....

File: src/main/java/stream/flarebot/flarebot/database/SQLController.java [hunk 2]
```diff
@@ -29,10 +28,8 @@
         dataSource.setPortNumber(3306);
         dataSource.setServerName("127.0.0.1");
         dataSource.setUser("flare");
-        dataSource.setURL(dataSource
-                .getURL() + "?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC&autoReconnect=true&useSSL=false");
-        dataSource.setUrl(dataSource
-                .getUrl() + "?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC&autoReconnect=true&useSSL=false");
+        dataSource.setURL(dataSource.getURL() + "?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC&autoReconnect=true&useSSL=false");
+        dataSource.setUrl(dataSource.getUrl() + "?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC&autoReconnect=true&useSSL=false");
```
Comment (ArsenArsen) [inline src/main/java/stream/flarebot/flarebot/database/SQLController.java]: Why do you people like changing unrelated files?

Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/database/SQLController.java]: That wasn't me. I don't think.

File: src/main/java/stream/flarebot/flarebot/commands/administrator/WelcomeCommand.java [hunk 3]
```diff
@@ -1,70 +1,155 @@
...
+                            MessageUtils.getUsage(this, channel, sender).queue();
+                        }
+                    } else {
+                        MessageUtils.getUsage(this, channel, sender).queue();
+                    }
                 }
-            } else if (args[0].equalsIgnoreCase("set")) {
-                if (guild.getWelcome() != null) {
+            } else if (args[0].equalsIgnoreCase("setchannel")) {
+                if (args.length == 1) {
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/administrator/WelcomeCommand.java]: This will always be true a sister you check args[0]

File: src/main/java/stream/flarebot/flarebot/commands/administrator/WelcomeCommand.java (hunk 1 above)
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/commands/administrator/WelcomeCommand.java]: for welcome message list.
